docs/documentation/documentation.html

```diff
@@ -59,8 +59,35 @@ <h1>Handleiding</h1>
           <section id="manon-quick-start">
             <h1>Manon gebruiken binnen een project.</h1>
             <p class="warning" role="group" aria-label="let op">
-              <span>Let op:</span> Aan deze instructies wordt nog gewerkt.
-            </p>
+                <span>Let op:</span> Aan deze instructies wordt nog gewerkt.
+              </p>
+          </section>
+          <section>
+              <h2>Manon installeren (lokaal docs bekijken)</h2>
+              <p>Manon maakt gebruik <a href="https://pnpm.io/installation">pnpm</a>, een package die beschikbaar is via npm.</p>
+              <ol>
+                <li>
+                    Manon downloaden: <code>git clone https://github.com/minvws/nl-rdo-manon.git</code>
+                </li>
+                <li>
+                    Open de folder waar Manon is geïnstalleerd, bijvoorbeeld: <code>cd /manon/install/folder</code>
+                </li>
+                <li>
+                    pnpm installeren: <code>npm install pnpm </code> 
```

Comment:
This install pnpm as a local dependency, rather than a globally available script. Depending on the user's setup that _might_ work (e.g. if they have `./node_modules/.bin` in their `PATH`, which is a fairly common "hack"), but is likely to lead to issues.

It's probably better to advise users to use `npm install -g pnpm` instead, or better yet, direct them to https://pnpm.io/installation for installation instructions.

Reply:
Ah right, fixed this by pointing the user to the official docs for install instructions.